vsts/pipelines/templates/_releaseJobTemplate.yml

```diff
@@ -51,7 +51,12 @@ jobs:
     inputs:
       artifactName: 'signed_binaries'
       path: $(Build.SourcesDirectory)/artifacts
-
+  
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
@cormacpayne I was primarily trying to keep component detection from being called twice within a single job. I think that it is ok here, as when it is invoked elsewhere in this file, it isn't within the same job. The issue we were running into was when it was called within [each test build job in the CI pipeline](https://github.com/microsoft/Oryx/blob/main/vsts/pipelines/ci.yml#L149-L153) since it is in `_setReleaseTag` AND `_buildTemplate`. The behavior I saw was that when called twice in the same job, only the first result was sent out, which ended up being a result from before the solution was built, which meant that it didn't actually detect the packages.

Reply:
Gotcha -- makes sense to me 👍